scripts/fix-code-style.js

```
@@ -0,0 +1,126 @@
+/**
+ * Fix Code Style Script
+ *
+ * This script runs all code style fixers in the correct order:
+ * 1. ESLint auto-fix for TypeScript issues
+ * 2. ESLint auto-fix for all issues
+ * 3. Prettier formatting
+ *
+ * It's designed to be used in CI environments to automatically fix code style issues.
+ */
+
+const { execSync } = require('child_process');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)